src/lib/testing/proposals.ts

```diff
@@ -31,6 +32,14 @@ export type ProposalEvaluationTestInput = Partial<Prisma.ProposalEvaluationCreat
     assignee: { group: ProposalSystemRole } | TargetPermissionGroup<'role' | 'user'>;
     operation: Extract<ProposalOperation, 'edit' | 'view' | 'move' | 'comment'>;
   }[];
+  voteSettings?: {
```

Comment:
this data model is managed in the webapp, it would be better to just use 'any' here. its not much benefit to have ts support here, when its not the actual interface we use anyway